draft-ietf-quic-transport.md

```diff
@@ -2941,10 +2941,20 @@ A packet detected by a receiver as a duplicate does not affect the receiver's
 local ECN codepoint counts; see ({{security-ecn}}) for relevant security
 concerns.
 
-If an endpoint receives a packet without an ECT or CE codepoint, it responds per
-{{processing-and-ack}} with an ACK frame.  If an endpoint does not have access
-to received ECN codepoints, it acknowledges received packets per
-{{processing-and-ack}} with an ACK frame.
+If an endpoint receives a QUIC packet without an ECT or CE codepoint, it
+responds per {{processing-and-ack}} with an ACK frame without increasing any ECN
+counters.  Similarly, if an endpoint does not have access to received ECN
+codepoints, it does not increase ECN counters.
+
+Coalesced packets (see {{packet-coalesce}}) mean that several packets can share
+the same IP header.  ECN counters count each QUIC packet, not the enclosing IP
+packet or UDP datagram.
+
+Each packet number space maintains separate acknowledgement state and separate
+ECN counters.  For example, if one each of an Initial, 0-RTT, Handshake, and
+1-RTT QUIC packet are coalesced, the corresponding counters for the Initial and
+Handshake packet number space will be incremented by one and the counters for
+the 1-RTT packet number space will be increased by two.
```

Comment:
Oh, well that's weird.

Reply:
Consistent with our mantra that coalesced packets are treated just as if each was received separately, though.